.github/SECURITY.md

```diff
@@ -0,0 +1,53 @@
+# Security Policy
+
+#### This is our policy for reporting security vulnerabilities and overall guidelines on what you should do upon discovering one!
+
+---
+
+<!--
+### Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+-->
+
+## Reporting Security Vulnerabilities
+
+<!--
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
+-->
+
+#### Please use the GitHub Security Advisory "Report a Vulnerability" tab!
+
+In order to report a security vulnerability, you can use GitHub's built-in tool which easily allows you to calculate an _attack vector/CVSS string_ or attribute to an existing [CVE](https://cve.org) code. This allows us to accurately calculate the severity and/or importance of preventing it.
+
+### Spotting secrets in code
+
+If you spot a secret in the code, please let us know by contacting us. This helps us quietly remove the vulnerability without letting others abuse it.
```

Comment:
```suggestion
If you spot a secret in the code, please let us know by [contacting us](mailto: itswin11@outlook.com). This helps us quietly remove the vulnerability without letting others abuse it.
```

Reply:
Done in https://github.com/itsWindows11/ThreadSharp/pull/11/commits/2fbaba6cb99ce9256ec28fe96f27b2caa3e9d46c